interfaces/seccomp/template.go

```diff
@@ -549,3 +549,15 @@ pwritev
 # of socket(), bind(), connect(), etc individually.
 socketcall
 `)
+
+// bindSyscallWorkaround specifies the bind() syscall for systems which
+// don't use AppArmor but only seccomp. In those cases the net package
+// from go uses bind for IPv4/IPv6 support detection when loaded. This
+// gets a problem for all hooks when they use snapctl. On systems which
+// use AppArmor the bind() call is properly mediated and the caller gets
+// an error back.
```

Comment:
> Not sure I understand the problem given this description. If we get an error when apparmor is enabled, that would mean the bind always fails. If that's the case, why are systems without apparmor special?

The difference is that AppArmor gives back an error to the caller of `bind()` where Seccomp doesn`t. So in scenarios where you have both AppArmor takes first and returns an error to the caller and Seccomp is never reached. When AppArmor isn't present the check goes directly to Seccomp which doesn't return and error but kills the caller. That makes the difference we're trying to fix here.

> Also, can we please stop using the term "force devmode" in that context? This sounds like a misnomer for "no apparmor support". 

I am open to that. Just reused "forced devmode" as it was already there and used elsewhere. What we should keep in mind is that we a term which refers to a mode which misses a LSM rather than calling out explicitly AppArmor as this could be SELinux in the future too.